src/elements/content-sidebar/DocGenSidebar/TagTree.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './DocGenSidebar.scss';
+import { JsonData } from './types';
+
+interface TagTreeProps {
+    data: JsonData | string[];
+    level?: number;
+}
+
+const TagTree = ({ data, level = 0 }: TagTreeProps) => {
+    if (Array.isArray(data)) {
```

Comment:
in the interface, you say that it can also be JsonData. Should ```Array.isArray(data)``` return false if `data` is JsonData?

Reply:
discussed